README.md

```diff
@@ -122,6 +122,8 @@ Then `curl "http://exporter:7979/probe?target=http://scrape_target:8080/test/dat
 ```
 {"time_diff": "1m35s","anotherVar": "something"}.
 ```
+## Disable Http Accept Header
+For disabeling Http Accept header you need set `disable_http_accept_header: true` in configuration file. Default Http Accept Header **enable**.
```

Comment:
Since this is nothing complex and will likely be a less used option, it is better to keep the description in example config file only. I don't think it needs to have it's own section or explanation in the main doc.